src/components/cart-card/CartCard.jsx

```diff
@@ -0,0 +1,37 @@
+const CartCard = ({ card }) => {
+   const discount = () => {
+      const discountPercent =
+         ((card.price - card.discountPrice) / card.price) * 100;
+      return Math.trunc(discountPercent);
+   };
+
+   return (
+      <div className="card horizontal">
+         <img className="card-img" src={card.imgUrl} alt="product" />
+         <div className="card-content">
+            <h6 className="font-semibold">{card.title}</h6>
+            <div className="price-wrapper">
+               <h5>{`₹ ${card.discountPrice}`}</h5>
+               <h6 className="strike-through">{`₹ ${card.price}`}</h6>
```

Comment:
Suggestion: Instead of adding Rupee sign, can use the toLocaleString to get the currency symbol.

Reply:
Will work on it. Thanks for your suggestion.